src/Compilers/CSharp/Portable/BoundTree/DecisionTree.cs

```diff
@@ -162,6 +162,7 @@ internal DecisionTree ComputeDecisionTree()
             }
 
             var result = DecisionTree.Create(_switchStatement.Expression, _switchStatement.Expression.Type, _enclosingSymbol);
+            var subsumptionTree = DecisionTree.Create(_switchStatement.Expression, _switchStatement.Expression.Type, _enclosingSymbol);
```

Comment:
We are maintaining separate decision trees for sybsumption checking (`subsumption`) and for code generation (`result`).

---
In reply to: [79474487](https://github.com/dotnet/roslyn/pull/13770#discussion_r79474487) [](ancestors = 79474487)